src/index.html

```diff
@@ -7,12 +7,69 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product"
+      data-qa="card"
+    >
+      <img
+        src="../src/images/imac.jpeg"
+        alt="imac"
+        class="product__image"
+      />
+      <div class="product__art">
+        <span>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
+      </div>
+      <div class="product__code">
+        <span>Product code: 195434</span>
+      </div>
```

Comment:
This div is redundant, fix it everywhere
```suggestion
      <p class="product__code">
        Product code: 195434
      </p>
```